angular/projects/spark-angular/src/lib/components/sprk-autocomplete/sprk-autocomplete.stories.ts

```diff
@@ -15,12 +16,21 @@ export default {
     SprkAutocompleteInputContainerDirective,
   },
   decorators: [
-    storyWrapper(
-      (storyContent) => `<div class="sprk-o-Box">${storyContent}<div>`,
+    moduleMetadata({
+      imports: [SprkAutocompleteModule],
+    }),
+    componentWrapperDecorator(
+      (story) => `<div class="sprk-o-Box">${story}</div>`,
     ),
   ],
   parameters: {
-    info: `
+    docs: {
+      source: {
+        type: 'code',
+      },
+      iframeHeight: 400,
```

Comment:
For some reason, this isn't being applied in the deploy preview.
<img width="1269" alt="Screen Shot 2021-06-30 at 12 08 04 PM" src="https://user-images.githubusercontent.com/15703773/123994911-d4e70400-d99b-11eb-884a-f229b3c7fb02.png">

Reply:
@guern1kn , it doesn't get applied with inline story rendering. I have a note up in the PR description about it. I left them in the code because it would be something we want to have and if SB adds it in a patch fix then we don't have to re-add it all.